scripts/api/data/metadatablocks/migration.tsv

```diff
@@ -0,0 +1,5 @@
+#metadataBlock	name	dataverseAlias	displayName	blockURI											

```

Comment:
Do we need this migration.tsv? It isn't loaded by default on new installations?

Reply:
It is not required and is not auto-loaded. It's a bit of a complex situation: The RDA grant had, as one goal, the idea of being able to import Bags created by other repos. An expectation there is that the other repo (a Dataverse with different blocks or some other repo software entirely) could have metadata that doesn't fit into the receiving Dataverse's schema (including blocks). So - the metadataOnOrig field is a json structured field that can store any/all metadata that doesn't match a known field. So a transfer can be done without losing metadata. That said, putting metadata in this field makes it less useful than if there were a matching field. The current code will ignore metadata that doesn't match if this block is not installed and will use it if it is. I can explain that, or remove the code from the PR. For migration cases, I think even this limited functionality could be useful, but I suspect that most people using the 'experimental' API here won't want to enable it.